schema_based_on_diagram.sql

```diff
@@ -0,0 +1,57 @@
+-- create medical database
+CREATE DATABASE medical

```

Comment:
- Please add indexes for columns that are mostly used. Something like below will do;
```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
CREATE INDEX ON invoice_items (treatment_id);
CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
CREATE INDEX ON medical_histories_has_treatments (treatment_id);
```